pkg/mysqlcluster/utils.go

```diff
@@ -60,3 +72,98 @@ func getStatusFromKVerb(verb kutil.VerbType) string {
 		return statusSkip
 	}
 }
+
+func getPodForHostname(client kubernetes.Interface, ns string, lbs map[string]string, hostname string) (*core.Pod, error) {
```

Comment:
What is lbs?

Reply:
`labels`, I can't name them `labels` because there is an import with the same name.